### ISSUE TITLE
I get empty specs and I'm not sure why...

### ISSUE THREAD

**Issue by virtualdogbert** [ePages-de/restdocs-api-spec#129]
So I followed the documentation, at least how I understood it and I was able to get 
openapi
openapi3
postman

running, but they generate  empty spec files my test looks like this:
```
import com.epages.restdocs.apispec.RestAssuredRestDocumentationWrapper
import geb.spock.GebSpec
import grails.gorm.transactions.Rollback
import grails.testing.mixin.integration.Integration
import groovy.json.JsonSlurper
import io.restassured.RestAssured
import io.restassured.builder.RequestSpecBuilder
import io.restassured.specification.RequestSpecification
import org.junit.Rule
import org.springframework.beans.factory.annotation.Value
import org.springframework.http.HttpStatus
import org.springframework.http.MediaType
import org.springframework.restdocs.JUnitRestDocumentation
import org.springframework.restdocs.payload.JsonFieldType
import org.springframework.restdocs.payload.PayloadDocumentation

import static org.springframework.restdocs.restassured3.RestAssuredRestDocumentation.documentationConfiguration

@Integration
@Rollback
class AppVersionControllerSpec extends GebSpec {

	static final String APP_VERSION_ENDPOINT = '/myapp/api/appVersion'

	@Value('${local.server.port}')
	protected int port

	@Rule
	protected JUnitRestDocumentation restDocumentation = new JUnitRestDocumentation('build/docs/generated-snippets')

	private RequestSpecification documentationSpec

	def setupSpec() {
	}

	def setup() {
		//set documentation specification
		this.documentationSpec = new RequestSpecBuilder()
			.addFilter(documentationConfiguration(this.restDocumentation))
			.build()
	}

	void "Test and document appVersion request (GET request, index action) to end-point: /api/appVersion"() {
		given: "A rest request for the App Version"
			RequestSpecification requestSpecification = RestAssured
				.given(this.documentationSpec)
				.accept(MediaType.APPLICATION_JSON_VALUE)
				.filter(
					RestAssuredRestDocumentationWrapper.document(
						'app-version-example',
						PayloadDocumentation.responseFields(
							PayloadDocumentation.fieldWithPath('status').ignored(),
							PayloadDocumentation.fieldWithPath('data').ignored(),
							PayloadDocumentation.fieldWithPath('data.version').type(JsonFieldType.STRING).description('The version of the Transition Manager'),
							)
					)
				)
		when: "GET request is made to end-point for the Application Version"
			def response = requestSpecification
				.when()
				.header('Accept-Version', '1.0')
				.accept(MediaType.APPLICATION_JSON_VALUE)
				.contentType(MediaType.APPLICATION_JSON_VALUE)
				.port(this.port)
				.get(APP_VERSION_ENDPOINT)

			def responseJsonObject = new JsonSlurper().parseText(response.body().asString())

		then: "status is OK"
			response.then()
					.assertThat()
					.statusCode(HttpStatus.OK.value())

			responseJsonObject instanceof Map

			responseJsonObject.status == 'success'
			responseJsonObject.data.version

	}

}
```

sample output:
```
{
  "swagger" : "2.0",
  "info" : {
    "description" : "My API description",
    "version" : "1.0.0",
    "title" : "My API"
  },
  "host" : "localhost:8080",
  "basePath" : "/api",
  "tags" : [ ],
  "schemes" : [ "http" ],
  "paths" : { },
  "definitions" : { }
}
```

Can you see anything I might be missing because this works in the sense that I can generate asciidoctor from it? In the config I did omit the:
` tagDescriptionsPropertiesFile = 'src/docs/tag-descriptions.yaml'`
as I didn't have the file. I tried adding it in, but I still don't get anything in my spec output.
Also I'm using version 0.9.7, of the gradle plugin with a Geb spock test running the RestAssured.

In my resource.json output I do have this, however:
```
{
  "operationId" : "app-version-example",
  "summary" : null,
  "description" : null,
  "privateResource" : false,
  "deprecated" : false,
  "request" : {
    "path" : "/myapp/api/appVersion",
    "method" : "GET",
    "contentType" : null,
    "schema" : null,
    "headers" : [ ],
    "pathParameters" : [ ],
    "requestParameters" : [ ],
    "requestFields" : [ ],
    "example" : null,
    "securityRequirements" : null
  },
  "response" : {
    "status" : 200,
    "contentType" : "application/json",
    "schema" : null,
    "headers" : [ ],
    "responseFields" : [ {
      "attributes" : { },
      "description" : "The version of the Transition Manager",
      "ignored" : false,
      "path" : "data.version",
      "type" : "STRING",
      "optional" : false
    } ],
    "example" : "{\"status\":\"success\",\"data\":{\"version\":\"Version 5.0.0-alpha (Development)\"}}"
  },
  "tags" : [ "myapp" ]
}
```

**Comment by ozscheyge** [comment]
Hey @virtualdogbert ,

I'm not using RestAssured so I don't know how that wrapper works exactly.

But judging by the `resource.json`, the intermediate "api-spec format" is generated correctly from what you document in your test.

What seems to fail, are the gradle plugins picking up this snippet and converting it to the desired target formats.

Mind posting your `build.gradle`?

**Comment by virtualdogbert** [comment]
I got it! I took another look at my build.gradle and the docs based on what you said, saw the config I copied from one of the examples, for one of the specs, I had snippetsDirectory and the others I didn't. further realized I had a leading slash, which is what was causing the issue.

Some days I can't see the forest through the trees, some systems want a leading slash and others don't, just didn't think of it at the time. In any case, thank you.
